client/data/agency-dashboard/use-sync-jetpack-connection-status.ts

```diff
@@ -0,0 +1,61 @@
+import { useQuery, useQueryClient } from '@tanstack/react-query';
+import { useContext, useEffect, useMemo } from 'react';
+import SitesOverviewContext from 'calypso/jetpack-cloud/sections/agency-dashboard/sites-overview/context';
+import { Site } from 'calypso/jetpack-cloud/sections/agency-dashboard/sites-overview/types';
+import wpcom from 'calypso/lib/wp';
+
+const useSyncJetpackConnectionStatus = (
```

Comment:
I'm curious about something regarding the naming choice, particularly, because this queries the Jetpack connection. Is there a reason to prefix it with "sync"?

Reply:
The query synchronizes the connection status with another query (this happened in the `onSuccess` and is now done in the `useEffect` call of the hook). The mere purpose of this custom hook is not just to query for it but also update the cache of the query which fetches the sites.